src/game/ObjectMgr.cpp

```diff
@@ -1975,6 +1975,38 @@ PetLevelInfo const* ObjectMgr::GetPetLevelInfo(uint32 creature_id, uint32 level)
     return &itr->second[level - 1];                         // data for level 1 stored in [0] array element, ...
 }
 
+PetFamilyStatMods const* ObjectMgr::GetPetFamilyStatMods(uint32 family) const
+{
+    // Loading levels data
+    //                                                   0      1     2     3
+    QueryResult* result  = WorldDatabase.Query("SELECT family, hmod, amod, dmod FROM pet_familystats");
+
```

Comment:
Nice, we should include that table in the mangos.sql and add an update to it.
About the query yes, its clearly kind af table that need to be preloaded.